.circleci/config.yml

```diff
@@ -104,6 +105,7 @@ jobs:
     steps:
       - checkout
       - setup_remote_docker
+      - sonarcloud/scan
```

Comment:
I see we're making the CCI pipeline itself run the Sonar Scan rather than depending on docker container to do that.

Together with the changes on Ilko's PR: [commit eca495](https://github.com/LBHackney-IT/direct-debit-submissions/commit/eca495daccaedc755912dc9012af3bd5242460a5). This will make it so the sonar scan gets run twice.

Are the changes highlighted here (ln108) are meant to replace the docker sonar scan run?